src/main/java/org/glassfish/concurro/cdi/ConcurrentCDIExtension.java

```diff
@@ -191,7 +191,6 @@ public <T> void processBean(@Observes ProcessBean<T> event) {
      */
     void afterBeanDiscovery(@Observes final AfterBeanDiscovery event, BeanManager beanManager) {
         try {
-            log.severe("afterBeanDiscovery");
```

Comment:
Perhaps for consistency, instead of removing it - it could follow the `beforeBeanDiscovery`, which is logged as finest and uses class name.

Reply:
I see. I've modified the code to follow the `beforeBeanDiscovery`. https://github.com/eclipse-ee4j/glassfish-concurro/pull/89/commits/32d5aaa9a322ddaabbcb8c7eeaa22ec7c0e6fcd4